src/domain/accounts/counterfactual-safes/entities/__tests__/counterfactual-safe.builder.ts

```diff
@@ -0,0 +1,20 @@
+import { IBuilder, Builder } from '@/__tests__/builder';
+import { CounterfactualSafe } from '@/domain/accounts/counterfactual-safes/entities/counterfactual-safe.entity';
+import { faker } from '@faker-js/faker';
+import { getAddress } from 'viem';
+
+export function counterfactualSafeBuilder(): IBuilder<CounterfactualSafe> {
```

Comment:
The domain entity `CounterfactualSafe` still uses snake_case syntax, but a forthcoming PR will restrict the usage of snake_case to the datasources layer, as discussed in https://github.com/safe-global/safe-client-gateway/pull/1773#discussion_r1684190135